BlueElectrum.js

```diff
@@ -153,11 +153,13 @@ module.exports.getTransactionsByAddress = async function(address) {
 
 module.exports.ping = async function() {
   try {
+    await wait(200); // TODO: For some reason sometimes Android not not load (infinity internet connection error)
```

Comment:
its not to simple hack :) ? need test ?

Reply:
Not sure about real device but for me using simulator it helped, both dev and prod versions. It doesn't need testing as it used only in saga and create simple delay that will not cause any issues